app/posts/register-of-training-providers/2025-10-10-exploring-how-we-log-changes-to-provider-partnership-accreditations.md

```diff
@@ -0,0 +1,115 @@
+
+![Screenshot showing the ‘Provider partnership accreditations updated’ activity on the provider activity log](provider-activity-item--partnership-updated.png "Screenshot showing the ‘Provider partnership accreditations updated’ activity on the provider activity log")
+
+### Provider partnership deleted
+
+When a user deletes a partnership, we show a summary card containing the:
+
+- accredited provider
+- training partner
+- linked accreditations
+
+![Screenshot showing the ‘Provider partnership deleted’ activity on the global activity log](global-activity-item--partnership-deleted.png "Screenshot showing the ‘Provider partnership deleted’ activity on the global activity log")
+
+![Screenshot showing the ‘Provider partnership deleted’ activity on the provider activity log](provider-activity-item--partnership-deleted.png "Screenshot showing the ‘Provider partnership deleted’ activity on the provider activity log")
+
+### Linking activity
+
+We link activity items to their corresponding provider pages. For example, if the activity relates to partnerships, we link the provider name to their partnerships list.
+
+We remove links to providers if they have been deleted from the register. We do not delete items from the activity log, as it is an immutable record.
+
+![Screenshot showing a deleted provider on the global activity log](global-activity-item--provider-deleted.png "Screenshot showing a deleted provider on the global activity log")
+
+![Screenshot showing a deleted provider on the ‘Provider partnership created’ activity item in the global activity log](global-activity-item--partnership-created-provider-deleted.png "Screenshot showing a deleted provider on the partnership created card in the global activity log")
+
+### Showing last updated information
+
+We show who last updated the provider and when they did the update. For example:
+
+> Last updated by Anne Wilson on 28 October 2025 at 2:55pm
+
+![Screenshot showing who last updated the provider and when they did the update](provider-activity--last-updated.png "Screenshot showing who last updated the provider and when they did the update")
+
+## Further considerations
+
+We considered some changes that we did not implement. These included:
+
+- linking activity items to the previous version
+- searching and filtering the activity logs
+- showing what changed in the activity item
+
+### Linking activity item to previous version
+
+If there have been multiple revisions, we will consider linking a revision to the previous and next revision where relevant.
+
+### Searching and filtering the activity logs
+
+Over time, the activity log will grow large. Whilst we already include pagination, it may be necessary to include search and filtering mechanisms to make it easier for users to find relevant information.
```

Comment:
How many years are we provision for?

Reply:
This is yet to be decided.

Reply:
Can we add a line for Further consideration please